beacon-chain/blockchain/receive_block.go

```diff
@@ -101,8 +101,8 @@ func (s *Service) ReceiveBlockNoPubsub(ctx context.Context, block *ethpb.BeaconB
 
 	processedBlkNoPubsub.Inc()
 
-	// We write the latest saved head root to a feed for consumption by other services.
-	s.headUpdatedFeed.Send(bytesutil.ToBytes32(headRoot))
+	// We write the latest saved head slot to a feed for consumption by other services.
+	s.headUpdatedFeed.Send(block.Slot)
```

Comment:
No need for the head root, all we want to know is a new canonical head was saved and its slot suffices